keyboards/handwired/dactyl_manuform/4x5/keymaps/ibnuda/keymap.c

```diff
@@ -0,0 +1,63 @@
+#include QMK_KEYBOARD_H
+
+#include "action_layer.h"
+#include "eeconfig.h"
```

Comment:
the `action_layer.h` shouldn't be needed, and I think the eeconfig.h isn't either, but not as sure on that one.